cmd/cbft/init_mem.go

```diff
@@ -33,6 +37,11 @@ func initMemOptions(options map[string]string) (err error) {
 		}
 		memQuota = uint64(fmq)
 	}
+	memoryLimit, err := getTotalMemStats()
+	if memoryLimit < memQuota {
+		memQuota = memoryLimit
```

Comment:
We should put some buffer here to avoid OOM kill once the memoryLimit is exceeded.
So, 

memQuota = .75 * memoryLimit?

Reply:
Thanks!
Sounds good, added in a recent commit.